src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldTransformers/ColorFieldTransformer.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react';
+import log from 'loglevel';
+import type {Node} from 'react';
+import type {FieldTransformer} from '../types';
+import colorFieldTransformerStyles from './colorFieldTransformer.scss';
+
+export default class ColorFieldTransformer implements FieldTransformer {
+    transform(value: *): Node {
+        //test si contain # et 3 ou 6 charateres
```

Comment:
would remove the comment as the regex below is quite ease to understand imo. also, personally i dont understand french 😇 

```suggestion
```